pdns/dnsrecords.hh

```diff
@@ -284,6 +284,19 @@ public:
   string d_key;
 };
 
+class CDNSKEYRecordContent : public DNSRecordContent
+{
```

Comment:
I tried using class `CDNSKEYRecordContent : public DNSKEYRecordContent`, but couldn't get it to work that fast, might retry :).